packages/flutter_tizen/lib/src/rendering/platform_view.dart

```diff
@@ -0,0 +1,187 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: public_member_api_docs
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/scheduler.dart';
+
+import '../services/platform_views.dart';
+
+/// Source: [_PlatformViewState] flutter/packages/flutter/lib/src/rendering/platform_view.dart
```

Comment:
We may use this standard format:
```suggestion
/// Source: [_PlatformViewState] in flutter/src/rendering/platform_view.dart
```
Notice that the path starts with `flutter/src` (without `lib` within it).